lightpath/ui/widgets.py

```diff
@@ -187,7 +193,8 @@ def update_state(self, *args, **kwargs):
         self.state_label.setText(self.last_state.name)
         color = state_colors[self.last_state.value]
         style_color = to_stylesheet_color(color)
-        self.state_label.setStyleSheet("QLabel {color: %s}" % style_color)
+        style_sheet = "QLabel {color: %s}" % style_color
```

Comment:
```suggestion
        style_sheet = f"QLabel {{color: {style_color} }}"
```

Reply:
I'm way too scared of the nested `}` here to put this in